### DISCUSSION TITLE
Mesh on wan on vlan

### DISCUSSION THREAD

**Discussion by albialbi** [freifunk-gluon/gluon#3331] [Q&A]
How can I do Mesh on vlan port on wan port?
Following does not work, what is wrong?
uci set gluon.iface_wan_vlan9=interface
uci set gluon.iface_wan_vlan9.name='eth0.9'
uci add_list gluon.iface_wan_vlan9.role='mesh'
uci commit gluon
gluon-reconfigure
reboot

**Comment by blocktrron** [comment]
Please post the full config output. You need to remove the pseudointerface from the default roleset if eth0 is enslaved in there.
